source/features/releases-tab.tsx

```diff
@@ -76,6 +76,9 @@ async function addReleasesTab(): Promise<false | void> {
 	// This re-triggers the overflow listener forcing it to also hide this tab if necessary #3347
 	repoNavigationBar.replaceWith(repoNavigationBar);
```

Comment:
I haven't verified whether this works or is necessary, but it's a tiny operation so let's leave it

Reply:
We can drop it, afaict it has no effect on the reflow whatsoever